src/utils/resources/vm/hooks/useOpenShiftConsoleDynamicPluginSDK.ts

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+import KubevirtPluginContext from 'src/views/virtualmachines/KubevirtPluginContext';
+
+export const useOpenShiftConsoleDynamicPluginSDK = () => {
+  const newContext = useContext(KubevirtPluginContext);
+  return newContext;
```

Comment:
So return the SDK here, not the whole context. This will allow you to unpack the contents of the SDK in the same way you would if you were handling the actual object post import.

For example, deconstructing time stamp should take one step: https://github.com/KevinFCormier/kubevirt-plugin/pull/1/files#diff-513bf2a84320f8f2596b63431d19efa0d3ed8b8ee7bd6b5fe938fd27e0b5b554R75-R76